app/src/ui/about/about.tsx

```diff
@@ -15,6 +15,7 @@ import { Loading } from '../lib/loading'
 import { RelativeTime } from '../relative-time'
 import { assertNever } from '../../lib/fatal-error'
 import { ReleaseNotesUri } from '../lib/releases'
+import { LinuxReleasesUri } from '../lib/releases'
```

Comment:
:art: this can be combined with the previous `import`

Reply:
@shiftkey - I've updated this in the new commit.  Thanks for the guidance and feedback!